terracumber_config/tf_files/SUSEManager-5.0-build-validation-paygo-AWS.tf

```diff
@@ -0,0 +1,498 @@
+}
+
+variable "AVAILABILITY_ZONE" {
+  type = string
+  default = null
+}
+
+variable "KEY_FILE" {
+  type = string
+  default = "/home/jenkins/.ssh/testing-suma.pem"
+}
+
+variable "KEY_NAME" {
+  type = string
+  default = "testing-suma"
+}
+
+variable "SERVER_REGISTRATION_CODE" {
+  type = string
+  default = null
+}
+
+variable "PROXY_REGISTRATION_CODE" {
+  type = string
+  default = null
+}
+
+variable "SLES_REGISTRATION_CODE" {
+  type = string
+  default = null
+}
+
+variable "ALLOWED_IPS" {
+  type = list(string)
+  default = []
+}
+
+variable "NAME_PREFIX" {
+  type = string
+  default = null
+}
+
+variable "SERVER_AMI" {
+  description = "Custom AMI ID to use for server. Leave empty to use the default SUSE Manager image."
+  type = string
+  default     = ""  # Default to empty string if not set
+}
+
+variable "PROXY_AMI" {
+  description = "Custom AMI ID to use for proxy. Leave empty to use the default SUSE Manager image."
+  type        = string
+  default     = ""  # Default to empty string if not set
+}
+
+variable "ARCHITECTURE" {
+  description = "Select Server architecture"
+  type        = string
+  default     = "x86_64"
+}
+
+provider "aws" {
+  region     = var.REGION
+}
+
+module "base" {
+  source                   = "./modules/base"
+  product_version          = "5.0-released"
+  name_prefix              = var.NAME_PREFIX
+  mirror                   = var.MIRROR
+  testsuite                = true
+  use_avahi                = false
+  use_eip_bastion          = false
+  is_server_paygo_instance = true
+  provider_settings        = {
+    availability_zone = var.AVAILABILITY_ZONE
+    region            = var.REGION
+    ssh_allowed_ips   = var.ALLOWED_IPS
+    key_name          = var.KEY_NAME
+    key_file          = var.KEY_FILE
+  }
+}
+
+module "mirror" {
+  source = "./modules/mirror"
+  base_configuration = module.base.configuration
+  disable_cron = true
+  provider_settings = {
+    public_instance = true
+  }
+  image = "opensuse155o"
+}
+
+module "server" {
+  source                     = "./modules/server"
+  base_configuration = merge(module.base.configuration,
+    {
+      mirror = null
+    })
+  name                       = "server"
+  product_version            = "5.0-paygo"
+  image                      = var.SERVER_AMI != "" ? var.SERVER_AMI : "suma-server-50-${var.ARCHITECTURE}-ltd-paygo"
+  main_disk_size             = 200
+  repository_disk_size       = 1500
+  database_disk_size         = 0
+
+  auto_accept                    = false
+  monitored                      = false
+  disable_firewall               = false
+  allow_postgres_connections     = false
+  skip_changelog_import          = false
+  create_first_user              = false
+  mgr_sync_autologin             = false
+  create_sample_channel          = false
+  create_sample_activation_key   = false
+  create_sample_bootstrap_script = false
+  publish_private_ssl_key        = false
+  use_os_released_updates        = true
```

Comment:
I would like to clarify this point, as we had a discussion some days ago where we agreed on the contrary.
Once we clarify, let's add an additional line into our confluence section for agreements about BV, if we need something different for AWS.

Reply:
More or less a copy paste of what we have for all the other BVs. Let's open a separate PR if we want to change this.

Reply:
A week ago we reach that agreement: https://github.com/uyuni-project/uyuni/pull/9540/files#r1877809363